src/APL/fileAPL.test.ts

```diff
@@ -0,0 +1,119 @@
+import { promises as fsPromises } from "fs";
+import { afterEach, describe, expect, it, vi } from "vitest";
+
+import { FileAPL } from "./fileAPL";
+
+const stubAuthData = {
+  domain: "example.com",
+  token: "example-token",
+};
```

Comment:
protip (here it doesn't matter) - it's safer in tests to ensure immutability (eg method `getStubAuthData()`) that will create a fresh one on every test. 

technically, inside the test, we can't ensure that some tested code doesn't mutate/modify stub/mock and this object can leak to other test with mutated value